### DISCUSSION TITLE
What's the best way to migrate from Hyas  to  Thulite?

### DISCUSSION THREAD

**Discussion by cnichte** [gethyas/hyas#493] [Q&A] [answered]
Hi! Will a remove of these with  `npm uninstall ...` ...

```
    "@hyas/doks-core": "^1.6.1",
    "@hyas/images": "^3.2.0",
    "@hyas/inline-svg": "^1.1.0",
    "@hyas/seo": "^2.3.0",
    "gethyas": "^2.4.2",
```

and replace them with their @thulite/... libs do the trick?

best regards Carsten

**Comment by h-enk** [comment, marked as answer]
Basically, yes. Plus, update the configuration.

See: [Upgrade to Doks v1](https://getdoks.org/upgrade-guides/upgrade-to-v1/)

**Reply by cnichte** [reply]
Thank you. Updating the project itself did not work. I got far too many errors. I have done a completely new install of thulite, in a fresh project, and then checked the data from the old project with a diff. Tool and copied the stuff over individually. That worked well.

**Reply by h-enk** [reply]
Glad to hear, you got it working (this way)!
